src/test/java/uk/gov/digital/ho/proving/income/api/FinancialStatusResourceTest.java

```diff
@@ -115,4 +123,24 @@ private FinancialStatusCheckResponse getResponse() {
         return new FinancialStatusCheckResponse(new ResponseStatus("100", "OK"), individuals, categoryChecks);
     }
 
+    @Test
+    public void shouldLogWhenRequestReceived() {
+        Logger rootLogger = (Logger) LoggerFactory.getLogger(FinancialStatusResource.class);
+        rootLogger.setLevel(Level.INFO);
+        rootLogger.addAppender(mockAppender);
+
+        when(mockNinoUtils.sanitise(realNino)).thenReturn(sanitisedNino);
+        when(mockNinoUtils.redact(sanitisedNino)).thenReturn(redactedNino);
+        when(mockHelper.calculateResponse(any(), any(), any())).thenReturn(getResponse());
+
+        service.getFinancialStatus(new FinancialStatusRequest(applicants, LocalDate.of(2019,01,01), 0 ));
+
+        verify(mockAppender).doAppend(argThat(argument -> {
+            LoggingEvent loggingEvent = (LoggingEvent) argument;
+
+            return loggingEvent.getFormattedMessage().equals("Financial status check request received for RedactedNino - applicationRaisedDate = 2019-01-01, dependents = 0") &&
+                ((ObjectAppendingMarker) loggingEvent.getArgumentArray()[3]).getFieldName().equals("event_id");
```

Comment:
Assert against the actual value of the log entry, not just the key. This is how we used to do it before discovering how to test against the value - you'll find it somewhere in the code already. Actually, I thought it was you that worked out how to do it!